src/main/java/uk/gov/di/ipv/persistence/item/UserIssuedCredentialsItem.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.di.ipv.persistence.item;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBDocument;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbBean;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbPartitionKey;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbSortKey;
+
+import java.time.LocalDate;
+
+@DynamoDbBean
+public class UserIssuedCredentialsItem {
+
+    private String sessionId;
+    private String credentialIssuer;
+    private String credential;
+    private LocalDate dateCreated;
+
+    @DynamoDbPartitionKey
+    public String getSessionId() {
```

Comment:
And this getter and the setter below should be
getIpvSessionId
setIpvSessionId